### DISCUSSION TITLE
Getting error "internal Analyzer Failure"

### DISCUSSION THREAD

**Discussion by pavlos1982** [Security-Onion-Solutions/securityonion#9908] [General]
when trying analyze a case under observable tabs, I get the error "internal Analyzer Failure" see screensho[t, can anyone help, thanks]
![Screenshot 2023-03-08 003427](https://user-images.githubusercontent.com/5363713/223547197-987c90ab-e04b-4350-b892-bfccdfa99c29.jpg)

**Comment by InfosecGoon** [comment]
It looks like you're trying to use the Spamhaus analyzer and it's not responding with useful information. Can you post the rest of the JSON response? Is it doing this for other IPs, or is 69.X.X.X just not in the Spamhaus database?

**Reply by pavlos1982** [reply]
Hi,

I have tested another IP that claims to be malware targetting one of my Mikrotik routers, again went to analyze it same result see the result with the JSON:

![Screenshot 2023-03-09 125614](https://user-images.githubusercontent.com/5363713/223971341-75c25bd9-54f3-4327-8be1-8d20e5d7dd06.jpg)

**Reply by pavlos1982** [reply]
I also got another error message that seems to be pointing to my internet DNS that all the routing has been done to access the internet see below:

![Screenshot 2023-03-09 130240](https://user-images.githubusercontent.com/5363713/223972854-c74d59a4-680b-4914-990d-e5d6d0458954.jpg)

**Reply by pavlos1982** [reply]
I have also tried to disable the EDR on the domain controller thinking it might be that blocking it but still same issue internal error

**Reply by InfosecGoon** [reply]
Definitely looks like an issue with the DNS server response you're getting for the Spamhaus query. Is there any kind of filtering or sinkholing of DNS lookups on your network that could be blocking the lookup?

**Comment by pavlos1982** [comment]
Hi, it is working for some other cases see below, but for some, it is not working, in relation to the DNS I am using CloudFlare DNS instead of my ISP, do you think this causes an issue? see the screenshot showing it is working
![Screenshot 2023-03-14 112000](https://user-images.githubusercontent.com/5363713/224925347-f6f64fcc-bf92-498e-babf-cb844f7d5cbe.jpg)

not working

![Screenshot 2023-03-14 112430](https://user-images.githubusercontent.com/5363713/224926125-a0233f53-2098-48db-bd39-b46daad5f98f.jpg)
